client/src/pages/SearchResult/SingleSearchResult.tsx

```diff
@@ -60,6 +69,18 @@ export function SingleSearchResultPage() {
   // default should be no, not favorited. use boolean type. True is selected.
   const [heart, setHeart] = useState<boolean>(false);
   const prevState = "";
+  // dollar
+  let dollarAPI = restaurantData.price_level;
+  console.log("dollarAPI: " + dollarAPI);
```

Comment:
Please remove `console.log`s that don't return errors before merging into develop.